src/v/redpanda/application.cc

```diff
@@ -2279,6 +2285,15 @@ void application::wire_up_and_start(::stop_signal& app_signal, bool test_mode) {
 
     audit_mgr.invoke_on_all(&security::audit::audit_log_manager::start).get();
 
+    if (!audit_mgr.local().report_redpanda_app_event(
```

Comment:
discussion point: I wonder how useful it is to check the value and throw on failure, since theres no reason why this should fail. Unless someone modified the audit tunables and placed 0 for the semaphore/size limits

Reply:
> Unless someone modified the audit tunables and placed 0 for the semaphore/size limits

We should probably set a lower limit on this to prevent that from happening.  I think if we had that, this situation would probably never happen.  I could even argue this is an assertable event rather than just throwing an exception since if something can't enqueue into the audit log _this early_ in the start up, something is very wrong.